src/main/java/io/cdap/e2e/utils/GcpClient.java

```diff
@@ -63,6 +64,16 @@ private static int executeQuery(String query) throws InterruptedException, IOExc
     return 0;
   }
 
+  public static String executeSelectQuery(String query) throws InterruptedException, IOException {
```

Comment:
We already have an executeQuery() method here. Why can't we use it instead of adding another method?

Reply:
Existing method parses query result into integer.  Can not be used for String output.

Reply:
Well, then we should look at changing the existing method to return object and convert the results to the expected data type.

Reply:
We will need to find all the places where existing method is used and make change.
Instead can we mark existing method deprecated?

Reply:
Why do we need string output here? It looks like it returns the same (no of rows present in the table) instead of actual data. If they are same, wondering why do we need it as string instead of integer?

Reply:
No. We are using new method to execute queries which returns string(actual data from table cell).

Reply:
It seems confusing. The only difference is data type, but the methods are `executeQuery` and `executeSelectQuery`. I agree that there should be one method that returns `Optional<String>` and any parsing can be done by caller or by a helper method with a clear name calling the main one

Reply:
Changed new executeSelectQuery() to return Optional <String>.
Calls to existing executeQuery() in e2e-tests project changed to new executeSelectQuery();
Existing method executeQuery() is public it is being used outside e2e-tests project as well. Marked it as Deprecated.